error-issue/src/types.ts

```diff
@@ -80,3 +80,58 @@ export interface ErrorReport {
   dailyOccurrences: number;
   stacktrace: string;
 }
+
+/**
+ * Types used in Stackdriver API requests.
+ * Note that other fields may be present, but only the ones relevant are
+ * included in the definitions below.
+ */
+export namespace Stackdriver {
+  interface TrackingIssue {
+    url: string;
+  }
+
+  export interface ErrorGroup {
+    name: string;
+    groupId: string;
+    trackingIssues?: Array<{
+      url: string;
+    }>;
```

Comment:
should this be type `Array<TrackingIssue>`?

Reply:
Whoops, it started out as that but I decided to drop the extra non-exported interface for simplicity/readability. Removed the `TrackingIssue` interface, thanks!